src/main/java/seedu/mindmymoney/helper/Functions.java

```diff
@@ -0,0 +1,17 @@
+package seedu.mindmymoney.helper;
+
+import seedu.mindmymoney.constants.Indexes;
+
+/**
+ * Separates the user input into the command and description for easy reference.
+ *
+ * @return String array of user input.
+ */
+public class Functions {
```

Comment:
Hmm I'm not sure if we need a Functions class... I think we can keep it first, if it comes in useful later on then we can keep it, if not then can just delete.

Reply:
Was thinking of having it to be like helper functions, so we dont have to rewrite same functions in multiple places